### PR TITLE
added doc comments to the whole project

### PR DESCRIPTION
**Pull request by BrandonOCasey**
fixed incorrect doc comments
fixed some bad formatting
fixed some weird code

### CODE REVIEW

**Comment by imbcmdth** [comment]
NICE. 🐶

File: src/decrypter/decrypter.js [hunk 1]
```diff
@@ -135,6 +142,20 @@ export class Decrypter {
...
   }
+
+  /**
+   * a getter for step the maximum number of bytes to process at one time
+   *
+   * @return {Number} the value of step 32000
+   */
+  static get STEP() {
+    // 4 * 8000;
+    return 32000;
```
Comment (dmlap) [inline src/decrypter/decrypter.js]: Why is this a getter?

File: src/master-playlist-controller.js [hunk 2]
```diff
@@ -29,6 +32,15 @@ const parseCodecs = function(codecs) {
   return result;
 };
 
+/**
+ * the master playlist controller controller all interactons
```
Comment (dmlap) [inline src/master-playlist-controller.js]: > controller controller

Not a blocker.

File: src/decrypter/decrypter.js (hunk 1 above)
Comment (BrandonOCasey) [inline src/decrypter/decrypter.js]: I don't think it should be, I am going to change it back to what it was before

**Pull request merged**